ocular_cli/src/commands/chains/show_default.rs

```diff
@@ -1 +1,45 @@
+use crate::{config, prelude::*};
+use abscissa_core::{Command, Runnable};
+use clap::Parser;
+use ocular::chain::info::ChainInfo;
+use serde::Deserialize;
+use std::{collections::HashMap, fs, path::Path, str};
 
+#[derive(Command, Debug, Parser)]
+pub struct ShowDefaultCmd {}
+
+#[derive(Deserialize)]
+struct Chains {
+    default_chain: String,
+    chains: Vec<ChainInfo>,
+}
```

Comment:
probably fine to have this here for now, but it should probably be defined in another module when implementing the `set_default` command

Reply:
Great!